src/tests/tests/lists.js

```diff
@@ -0,0 +1,13 @@
+(function() {
+  QUnit.module('Test lists');
+  let mdfp = require('multiDomainFP');
+
+  QUnit.test('Check MDFP domains are all base domains', (assert) => {
+    for (var group of mdfp.multiDomainFirstPartiesArray) {
+      for (var domain of group) {
+        assert.ok(window.getBaseDomain(domain) == domain, 'is a basedomain');
+        assert.notOk(domain in window.publicSuffixes, 'not a public suffix');
```

Comment:
I'm not sure it makes sense to check against `window.publicSuffixes` directly (since the numeric values in that object alter the meaning of the suffixes, I think). Using the public suffix "API", `getBaseDomain` should get around low-level implementation details (https://github.com/EFForg/privacybadger/pull/1550#issuecomment-320357532).

Reply:
I changed the check to be `getBaseDomain('fakesubdomain.' + domain) == domain`. This should catch all cases where a basedomain is a suffix of `domain`. I *think* it should catch all cases when `domain` is a suffix of a basedomain.